internal/command/command.go

```diff
@@ -548,7 +548,7 @@ func appConfigFilePaths(ctx context.Context) (paths []string) {
 	return
 }
 
-var errRequireAppName = fmt.Errorf("we couldn't find a fly.toml nor an app specified by the -a flag. If you want to launch a new app, use '%s launch'", buildinfo.Name())
+var errRequireAppName = fmt.Errorf("the config for your app is missing an app name, add an app_name field to the fly.toml file or specify with the -a flag`")
```

Comment:
We don't have to change the error message here. Do you have reasons to not mention `fly launch`?

Reply:
Oh, that's just because #2009 recently adjusted the wording for this error message. I'm just transferring the new message from a custom error in `RequireAppName` to `errRequireAppName`, which had the old wording and was now unused.